data/brands/amenity/fast_food.json

```diff
@@ -617,6 +617,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Brezelkönig",
+      "id": "brezelkonig-096f6f",
+      "locationSet": {"include": ["at", "ch", "de"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Brezelkönig",
+        "brand:wikidata": "Q111728604",
+        "cuisine": "pretzel",
+        "diet:vegetarian": "yes",
+        "drink:espresso": "takeaway",
```

Comment:
I have same question with https://github.com/osmlab/name-suggestion-index/pull/6501#discussion_r859334678

Is this necessary?

Reply:
Removed.